### ISSUE TITLE
How to filter null value

### ISSUE THREAD

**Issue by gech4me** [tailflow/laravel-orion#59]
How to filters null value, alternative for the following query.

```
$users = User::where('active','=', null)->get()
$users = User::whereNotNull('active')->get();

```

**Comment by gech4me** [comment]
> Where do you want to apply the filter?
> 
> If it is for **index** query, then you can add the following to your controller:
> 
> ```
> protected function buildIndexFetchQuery(Request $request, array $requestedRelations): Builder
> {
>      $query = parent::buildIndexFetchQuery($request, $requestedRelations);
> 
>      $query->whereNotNull('active');
> 
>       return $query;
> }
> ```
> 
> https://tailflow.github.io/laravel-orion-docs/guide/models.html#customizing-queries

Yeah, but I want toggle active from the front-end.
```
  "filters" : [
        {"field" : "active", "operator" : "!=", "value" : null}
    ]
```
but there is a validation error. 
Thanks

**Comment by SachinGanesh** [comment]
This issue is fixed in v1.2.4 https://github.com/tailflow/laravel-orion/releases/tag/1.2.4

Duplicate of https://github.com/tailflow/laravel-orion/issues/50
